src/Domain/Creators/Shell.cpp

```diff
@@ -89,15 +84,44 @@ Shell::Shell(
     PARSE_ERROR(context,
                 "Cannot have periodic boundary conditions with a shell");
   }
+  if (not std::is_sorted(radial_partitioning_.begin(),
+                         radial_partitioning_.end())) {
+    PARSE_ERROR(context,
+                "Specify radial partitioning in ascending order. Specified "
+                "radial partitioning is: "
+                    << get_output(radial_partitioning_));
```

Comment:
Now you don't need the get_output or std::to_string anymore :)

Reply:
`error: invalid operands to binary expression ('basic_ostream<char, std::char_traits<char> >' and 'std::vector<double>')`

where is this overload?

Reply:
Utilities/StdHelpers.hpp